pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -815,6 +815,14 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					PropertyName: "chatid",
 					Required:     true,
 				},
+				{
+					Label:        "Message Thread ID",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Description:  "Integer Telegram Message Thread Identifier",
+					PropertyName: "messagethreadid",
+					Required:     false,
+				},
```

Comment:
let's add `ValidationRule`, a regular expression that makes sure that the input is an integer. Something like `^-?[0-9]{19}$`

Reply:
Done 👍